src/schemas/json/azure-iot-edge-deployment-2.0.json

```diff
@@ -256,11 +256,41 @@
                                     "type": "object",
                                     "patternProperties": {
                                         "^[^\\.\\$# ]+$": {
-                                            "type": "string",
-                                            "examples": [
-                                                "FROM /* INTO $upstream"
-                                            ],
-                                            "pattern": "^.+$"
+                                            "anyOf": [
+                                                {
+                                                    "type": "object",
+                                                    "required": [
+                                                        "route"
+                                                    ],
+                                                    "properties": {
+                                                        "route": {
+                                                            "type": "string",
+                                                            "examples": [
+                                                                "FROM /* INTO $upstream"
+                                                            ],
+                                                            "pattern": "^.+$"
+                                                        },
+                                                        "priority": {
+                                                            "type": "integer",
+                                                            "minimum": 0,
+                                                            "maximum": 4294967295
```

Comment:
Max value for the Priority is going to be 9 initially. (We haven't made the change in the iotedge codebase yet, but will be making that soon).

Reply:
Thanks, I updated it